src/ports/postgres/modules/graph/wcc.py_in

```diff
@@ -294,7 +294,7 @@ def wcc(schema_madlib, vertex_table, vertex_id, edge_table, edge_args,
     if not is_platform_pg():
         # Drop intermediate table created for Greenplum
         plpy.execute("DROP TABLE IF EXISTS {0}".format(edge_inverse))
-    plpy.execute("ALTER TABLE {0} RENAME TO {1}".format(newupdate, out_table))
+    plpy.execute("CREATE TABLE {0} AS SELECT * FROM {1}".format(out_table, newupdate))
```

Comment:
`newupdate` is dropped at the end of the function(line 316-317). We need this change because postgresql doesn't allow altering the schema of a table. If the user wants the output table in a specific schema, alter table line will fail.